### ISSUE TITLE
Reply to specific messages

### ISSUE THREAD

**Issue by baseljd** [bbernhard/signal-cli-rest-api#261]
### Feature Request

Are we able to reply to specific messages using the `/send` endpoint? Maybe sending the `message id` we would like to reply to along with the message.

Is it possible to support this?

**Comment by kevincw01** [comment]
I don't think so.  What I see when I receive a reply to a specific message is that the dataMessage node contains an additional json node of 'quote'.  Example below.  Looks like this needs to be implemented.

```
"dataMessage": {
      "timestamp": 1667352320729,
      "message": "Reply to you",
      "expiresInSeconds": 0,
      "viewOnce": false,
      "quote": {
        "id": <redacted>,
        "author": <redacted>,
        "authorNumber": <redacted>,
        "authorUuid": <redacted>,
        "text": "false",
        "attachments": []
      }
    }
```

**Comment by kevincw01** [comment]
The upstream signal-cli GitHub does support it so its feasible to update this github source to support:

``
#[rpc(name = "send", params = "named")]
    fn send(
        &self,
        account: Option<String>,
        recipients: Vec<String>,
        #[allow(non_snake_case)] groupIds: Vec<String>,
        #[allow(non_snake_case)] noteToSelf: bool,
        #[allow(non_snake_case)] endSession: bool,
        message: String,
        attachments: Vec<String>,
        mentions: Vec<String>,
        #[allow(non_snake_case)] quoteTimestamp: Option<u64>,
        #[allow(non_snake_case)] quoteAuthor: Option<String>,
        #[allow(non_snake_case)] quoteMessage: Option<String>,
        #[allow(non_snake_case)] quoteMention: Vec<String>,
        sticker: Option<String>,
        #[allow(non_snake_case)] storyTimestamp: Option<u64>,
        #[allow(non_snake_case)] storyAuthor: Option<String>,
    ) -> Result<Value>;
```
